### DISCUSSION TITLE
Welcome to flippermaker.github.io Discussions!

### DISCUSSION THREAD

**Discussion by natehome** [FlipperMaker/flippermaker.github.io#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Request features
  * Share ideas.
  * Engage with other community members.

**Comment by Unmanto** [comment]
hola soy nuevo y pido perdón por mis errores con las descargas y es que no me salia bien gracias y perdonar
